src/main/java/appeng/fluids/client/gui/GuiFluidTerminal.java

```diff
@@ -47,15 +47,15 @@
 import appeng.client.me.FluidRepo;
 import appeng.client.me.SlotFluidME;
 import appeng.client.me.InternalFluidSlotME;

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Wrong order for 'appeng.client.me.InternalFluidSlotME' import. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.imports.ImportOrderCheck)